docs/network-flow-visibility.md

```diff
@@ -35,6 +35,9 @@
     - [Output Flow Records](#output-flow-records)
   - [Grafana Flow Collector (migrated)](#grafana-flow-collector-migrated)
   - [ELK Flow Collector (removed)](#elk-flow-collector-removed)
+- [Layer 7 network flow export](#layer-7-network-flow-export)
```

Comment:
Other first level sections capitalize the title, so it should be "Layer 7 Network Flow Export" to be consistent.